### DISCUSSION TITLE
Special characters and FB

### DISCUSSION THREAD

**Discussion by KonradSotniczuk** [sherlock-project/sherlock#955] [Q&A]
Hi,
Thanks for your hard work!

After using it for a while, I have 2 questions.

1. If I want to use European special symbols, this will be enough - https://www.fon.hum.uva.nl/praat/manual/Special_symbols.html  
Like python sherlock 'XXXXX EEEi\eXXX'   ?   
I get some search results when I try it, and later an Error:
Traceback (most recent call last):
  File "C:\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\x\sherlock\sherlock\__main__.py", line 30, in <module>
    sherlock.main()
  File "D:\x\sherlock\sherlock\sherlock.py", line 630, in main
    with open(result_file, "w", encoding="utf-8") as file:
FileNotFoundError: [Errno 2] No such file or directory: 'XXXXXX EEEi\eXXX.txt'

2. For some searches it finds FB links, on others It does not. Not sure if it's a bug or I don;t understand how to use this.

**Comment by hoadlck** [comment]
I looked into it.  Windows does not allow `\` in a file name (probably because it is reserved for the directory separator).  So, when the results text file is attempted to be created, it fails.

I don't think that there are any sites that allow `\` in a user name.  You are only using it here because you are trying to enter a non-ASCII escape code.  Can you paste the actual utf-8 character on the console?  Not using any escape codes?  This was working for me.

In reality, Sherlock is expecting the username it is getting passed to be in utf-8 encoding.  It is not honoring the escape codes.
